### DISCUSSION TITLE
How to make an mlx array containing every number from 0-100 and skipping every Nth number

### DISCUSSION THREAD

**Discussion by kyrollosyanny** [ml-explore/mlx#1554] [Q&A] [answered]
Hello, I have a numpy code that makes a 1D array containing numbers from 0-100 while skipping every N'th element. This array will later be used to filter out data from an mlx array (basically removing every N'th column from the mlx array while also keeping the 0 element). I can make the indices in numpy as shown below, convert to an mlx array then use them select columns from the second array. But I was wondering if there is a way to make the indices in mlx to begin with. 

```
## any way to make conver this code to mlx?
import numpy as np
idx=np.arange(100)
idx_mask=np.mod(idx,4)!=0
idx_mask[0]=True
idx=idx[idx_mask]
print(idx)
```

Thank you so much

**Comment by angeloskath** [comment, marked as answer]
Haven't tested it but something like the following should work 

```
indices = mx.arange(100).reshape(-1, 4)
indices = indices[:, 1:].reshape(-1)
indices = mx.concatenate([mx.zeros((1,), type=mx.int32), indices])
```

**Reply by kyrollosyanny** [reply]
Thank you so much! that works so well!!
